include/wil/token_helpers.h

```diff
@@ -438,6 +438,63 @@ namespace wil
     }
 #endif // WIL_ENABLE_EXCEPTIONS
 
+    namespace details
+    {
+        template<size_t AuthorityCount> struct static_sid_t
+        {
+            BYTE Revision;
+            BYTE SubAuthorityCount;
+            SID_IDENTIFIER_AUTHORITY IdentifierAuthority;
+            DWORD SubAuthority[AuthorityCount];
+
+            PSID get()

```

Comment:
given `PSID` is an alias for `void*` it is dangerous. nothing we can do about it here that I'm aware of, but FYI.